llvm/lib/IR/DebugInfoMetadata.cpp

```diff
@@ -2011,6 +2010,347 @@ DIExpression::constantFold(const ConstantInt *CI) {
           ConstantInt::get(getContext(), NewInt)};
 }
 
+// Returns true if the Op is a DW_OP_constu.
+static bool isConstantVal(uint64_t Op) { return Op == dwarf::DW_OP_constu; }
+
+// Returns true if an operation and operand result in a No Op.
+static bool isNeutralElement(uint64_t Op, uint64_t Val) {
```

Comment:
could the first type be dwarf::LocationAtom?

Reply:
Yes but what is the difference/advantage of that?

Reply:
So it's less easy to confuse the two `uint64_t` parameters. It's a very minor improvement.